components/Collapse.tsx

```diff
@@ -0,0 +1,17 @@
+import { FC } from 'react'
+
+export interface CollapseProps {
+  title: string
+  children?: JSX.Element
+}
+
+const Collapse: FC<CollapseProps> = ({ title, children }) => {
+  return (
+    <details className="max-w-prose border p-3 mb-3 rounded">
+      <summary className="text-blue-light">{title}</summary>
```

Comment:
```suggestion
      <summary className="text-blue-light hover:underline hover:text-link-selected cursor-pointer focus:text-link-selected focus:underline">{title}</summary>
```

Following the design seen [here](https://design.canada.ca/common-design-patterns/collapsible-content.html), there should be hover and focus styling on the summary element